lib/fluent/plugin/out_tdlog.rb

```diff
@@ -22,7 +22,8 @@ class TreasureDataLogOutput < Output
     config_param :table, :string, :default => nil
     config_param :use_gzip_command, :bool, :default => false
 
-    config_param :endpoint, :string, :default => TreasureData::API::NEW_DEFAULT_ENDPOINT
```

Comment:
> This change doesn't break compatibilities

I think it's breaking change because parameter name is changed. Is it acceptable? (I don't know this repository's policy)

Reply:
The newer parameter is defined with `:alias => :endpoint` for compatibility.

Reply:
I see. Then, no problem.